packages/components/psammead-section-divider/src/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+import { boolean, shape, string } from 'prop-types';
+import { scriptPropTypes } from '@bbc/gel-foundations/prop-types';
+import {
+  getDoublePica,
+  GEL_FF_REITH_SANS,
+} from '@bbc/gel-foundations/typography';
+import { C_CLOUD_LIGHT, C_EBON } from '@bbc/psammead-styles/colours';
+
+const SectionTitle = styled.h2`
+  ${props => getDoublePica(props.script)};
+  color: ${C_EBON};
+  font-family: ${GEL_FF_REITH_SANS};
+
+  html:not([dir='rtl']) & {
+    margin-left: 8px;
+  }
+
+  html[dir='rtl'] & {
+    margin-right: 8px;
+  }
+`;
+
+SectionTitle.propTypes = {
+  script: shape(scriptPropTypes).isRequired,
+};
+
+const SectionRule = styled.div`
+  background: ${C_CLOUD_LIGHT};
+  height: 1px;
+  border: none;
+  flex-grow: 100;
+  margin: ${props => (props.inline ? 'auto 8px' : '8px')};
+`;
+
+const SectionDividerWrapper = styled.div`
```

Comment:
Hey, have you perhaps considered a more simpler (imo) solution that doesn't use flex? Ive made https://codepen.io/drewmcmillan/pen/OGOzLR which mirrors the news frontpage implementation at the moment, curious of your thoughts